frontend/src/components/activePage/TableInstance/TableInstance.tsx

```diff
@@ -54,6 +57,38 @@ const TableInstance: FC<ITableInstanceProps> = ({ ...props }) => {
 
   return (
     <>
+      <ModalAlert
+        headTitle="Destroy All"
```

Comment:
```suggestion
        headTitle="Confirm Instances deletion"
```

Reply:
This modification have been deleted because is part of a @cannarelladev's PR